NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java

```diff
@@ -390,4 +391,11 @@ private static void enableProtocolShim() {
     private static void instantPlayInUseEntity() {
         instantPlayInUseEntity = getBoolean("settings.instant-interaction", false);
     }
+
+    public static boolean enableImprovedHitReg;
+
+    private static void enableImprovedHitReg() {
+        enableImprovedHitReg = getBoolean("settings.use-improved-hitreg", false);
```

Comment:
I like how you have it disabled by default, It is a pretty high risk change, nice job

Reply:
Yeah this hitreg can be very different from what people are used to so I thought not everyone would be interested in having it on